srcpkgs/coin3/patches/disable-check_cxx_source_compiles.patch

```diff
@@ -0,0 +1,20 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -586,11 +586,11 @@ if(NOT (HAVE_WGL OR HAVE_AGL OR HAVE_CGL
+     list(APPEND COIN_TARGET_LINK_LIBRARIES OpenGL::GLX)
+   endif()
+   set(CMAKE_REQUIRED_LIBRARIES ${COIN_TARGET_LINK_LIBRARIES})
+-  check_cxx_source_compiles("
+-    #include <GL/gl.h>
+-    #include <GL/glx.h>
+-    int main() { (void)glXChooseVisual(0L, 0, 0L); glEnd(); return 0; }
+-  " HAVE_GLX)
++#  check_cxx_source_compiles("
++#    #include <GL/gl.h>
++#    #include <GL/glx.h>
++#    int main() { (void)glXChooseVisual(0L, 0, 0L); glEnd(); return 0; }
++#  " HAVE_GLX)
```

Comment:
This is assuming platform doesn't `HAVE_GLX`
I don't know anything about coin3 or freecad, but commit message is bogus.

Reply:
any suggestions I should change to make the commit better?

Reply:
> This is assuming platform doesn't `HAVE_GLX`

Yes. Coin works without glx for FreeCAD probably because the functionality is not needed by FreeCAD and Coin has a dummy implementation.